server/traces/time_converter.go

```diff
@@ -0,0 +1,65 @@
+package traces
+
+import (
+	"fmt"
+	"math"
+)
+
+const (
+	NANOSECOND_SCALE  = 0
+	MICROSECOND_SCALE = 1
+	MILLISECOND_SCALE = 2
```

Comment:
Those are not used, I left them here to make it easier to understand what those numbers represent